Source/msg.h

```diff
@@ -7,6 +7,8 @@
 
 #include <stdint.h>
 
+#include "quests.h"

```

Comment:
Any specific reason for leaving the blank line between the includes? Wouldn't this be preferred?
 ```suggestion
#include <stdint.h>
#include "quests.h"
```

Reply:
i think its commonly done so that system libs come first and aren't  accidentally sorted lower when formatting. But this might just be an issue stemming from our headers generally not doing a good job of stating dependencies.